spdmlib/src/common/mod.rs

```diff
@@ -739,6 +740,7 @@ pub struct SpdmRuntimeInfo {
     pub message_c: ManagedBuffer,
     pub message_m: ManagedBuffer,
     pub message_vca: ManagedBuffer,
+    pub content_changed: bool, // used by responder, when content changed and spdm version is 1.2, set to true.
```

Comment:
we have 3 states: not capable, changed, not changed.

Reply:
> we have 3 states: not capable, changed, not changed.

Updated. Still keep the type of content_changed in runtime info as bool, and changed the type of content_changed in SpdmMeasurementsResponsePayload struct to u8.